lib/awful/hotkeys_popup/keys/vim.lua

```diff
@@ -9,12 +9,12 @@
 
 local hotkeys_popup = require("awful.hotkeys_popup.widget")
 
-local vim_rule = {name="vim"}
+local vim_rule_any = {name={"vim", "VIM"}}
```

Comment:
Could this use some globbing, matching the start or end?

For me a vim instance looks like this:

> WM_NAME(COMPOUND_TEXT) = "✐ [No Name] (~/src/awesome) - vim"